### PR TITLE
refactor: Skip license checks for development

### PR DESCRIPTION
**Pull request by zomars**
## What does this PR do?

There has been a lot of confusion from people testing out locally, we should skip license check on local development although we should display a warning when people test out enterprise features.

image

Added console warning as well:

image

The banner should not be so intrusive but still needs to be visible enough to not remain unnoticed.

## Type of change

- Chore (refactoring code, technical debt, workflow improvements)
- This change requires a documentation update

## How should this be tested?

- [ ] On `yarn dev` without a valid license, you should be able to see workflows for example.
- [ ] On `yarn build && yarn start` you should see the normal license check.

## Mandatory Tasks
- [x] Make sure you have self-reviewed the code. A decent size PR without self-review might be rejected.

### CODE REVIEW

File: packages/features/ee/common/components/LicenseRequired.tsx [hunk 1]
```diff
@@ -25,6 +26,28 @@ const LicenseRequired = ({ children, as = "", ...rest }: LicenseRequiredProps) =
     <Component {...rest}>
       {hasValidLicense === null || hasValidLicense ? (
         children
+      ) : process.env.NODE_ENV === "development" ? (
+        /** We only show a warning in development mode, but allow the feature to be displayed for development/testing purposes */
```
Comment (zomars) [inline packages/features/ee/common/components/LicenseRequired.tsx:30]: Hopefully it is clear enough

File: packages/features/ee/common/components/LicenseRequired.tsx [hunk 2]
```diff
@@ -25,6 +26,28 @@ const LicenseRequired = ({ children, as = "", ...rest }: LicenseRequiredProps) =
...
+        <>
+          <TopBanner
+            text=""
```
Comment (zomars) [inline packages/features/ee/common/components/LicenseRequired.tsx:33]: Intentional to comply with required props. But not really needed.

**Review by ericrommel — APPROVED** [review]
Worked fine. Thanks...

image

**Comment by PeerRich** [comment]
great change!

**Pull request merged**